ecms_base/modules/custom/ecms_health_file_migration/config/install/migrate_plus.migration.health_file_json_to_file.yml

```diff
@@ -7,27 +7,29 @@ destination:
   plugin: 'entity:file'
 
 source:
-  constants:
-    DRUPAL_FILE_DIRECTORY: 'public://'
-    DIRECTORY_SEPERATOR: '/'
   plugin: url
   data_fetcher_plugin: http
+  urls:
+    - 'https://health.ri.gov/rss/pubs/publications-total-en.json.php'
   data_parser_plugin: json
-  # The JSON source file URL for the migration.
-  urls: 'https://health.ri.gov/rss/pubs/publications-media-en.json.php'
-  # Under 'fields', we list the data items to be imported. The first level keys
-  # are the source field names we want to populate (the names to be used as
-  # sources in the process configuration below). For each field we're importing,
-  # we provide a label (optional - this is for display in migration tools) and
-  # an selector (xpath) for retrieving that value. It's important to note that this xpath
-  # is relative to the elements retrieved by item_selector.
-
   item_selector: '[*]'
   fields:
     -
       name: id
-      label: 'ID'
+      label: ID
       selector: id
+    -
+      name: entity
+      label: Entity
+      selector: entity
+    -
+      name: topic
+      label: Topic
+      selector: topic
+    -
```

Comment:
Were new fields being added intentionally?

Reply:
No. But these fields were part of the source I was using as a template. I assumed that if the fields were needed when this source was used elsewhere then they would be needed here. Shouldn't the same source use the same fields?

Reply:
I have reverted everything except changing  the source URL